src/screens/OrgList/OrgList.tsx

```diff
@@ -319,7 +322,13 @@ function OrgList(): JSX.Element {
                 className={styles.cancel}
                 data-testid="closeOrganizationModal"
               >
-                <i className="fa fa-times"></i>
+                <img
```

Comment:
Why was the icon removed? The image would be an unnecessary overhead.

Reply:
It seems that despite the code being present with `i` tag, the icon is not displaying on the page. I've already tried using 5 different icons from Font Awesome, but the issue still persists.

The image is an SVG, which is lightweight and shouldn't add unnecessary overhead to the page.

Reply:
Each version of font-awesome has different names for its free icons. A free icon in v3 may not not be available in v4 and will therefore not display. Check the CSS for available icons to use and use the font-awesome website to see which ones are available for each version..

Reply:
@Rumone  
@palisadoes 

I addressed the request changes. Let me know if there's anything else to be done. Thanks